IPNI.md

```diff
@@ -470,11 +470,19 @@ Implementers are free to return results as they are found.
 
 The alternative routing systems currently supported is:
  * `ipfs-dht`: equivalent to searching records on the IPFS network.
+ * `legacy`: searches records via BitSwap broadcast over a set of peered nodes; see [Legacy Lookup Cascade](#legacy-lookup-cascade).
```

Comment:
:thought_balloon: a meta-comment on assuming things will never change ;-) 

`legacy` is a bit unfortunate name, because in 2 years something else will be considered "legacy".

Something like `bitswap-peering` would be way better and more future-proof.

Reply:
The term legacy is there to convey that this functionality will be deprecated.

Reply:
Deprecation could be mentioned in a comment?

If the routing method is bitswap peering, it makes sense to call it that since then you need no other context to know what it is.

Reply:
Thank you for pointing this out; deprecation notice added here: https://github.com/ipni/specs/pull/15

In terms of naming the one thing i wanted all of this to convey is that cascading over bitswap over peering is not here to stay and is "legacy" already.